rest/blip_client_test.go

```diff
@@ -716,3 +728,32 @@ func (btr *BlipTesterReplicator) storeMessage(msg *blip.Message) {
 	defer btr.messagesLock.Unlock()
 	btr.messages[msg.SerialNumber()] = msg
 }
+
+func (btc *BlipTesterClient) WaitForBlipMessage(docId, revId string) (msg *blip.Message, found bool) {
```

Comment:
A small rename to make this clearer it's specifically for a rev message.

```suggestion
func (btc *BlipTesterClient) WaitForBlipRevMessage(docId, revId string) (msg *blip.Message, found bool) {
```

Reply:
Fixed in latest revision.